uber/templates/jobs/everywhere.html

```diff
@@ -1,10 +1,85 @@
 {% extends "base-admin.html" %}
```

Comment:
This page now renders everything by Javascript, so when we make an assignment, we don't reload the full page, which is still too slow to be convenient.